conjur/argument_parser/_role_parser.py

```diff
@@ -0,0 +1,84 @@
+"""
+Module For the RoleParser
+"""
+import argparse
+from conjur.argument_parser.parser_utils import command_description, command_epilog, formatter, \
+    title_formatter
+from conjur.wrapper.argparse_wrapper import ArgparseWrapper
+
+
+# pylint: disable=too-few-public-methods
+class RoleParser:
+    """Partial class of the ArgParseBuilder.
+    This class add the Role subparser to the ArgParseBuilder parser."""
+
+    def __init__(self):
+        self.resource_subparsers = None  # here to reduce warnings on resource_subparsers not exist
+        raise NotImplementedError("this is partial class of ArgParseBuilder")
+
+    def add_role_parser(self):
+        """
+        Method adds role parser functionality to parser
+        """
+        role_parser = self._create_role_parser()
+        role_subparser = role_parser.add_subparsers(title="Subcommand", dest='action')
+
+        self._add_role_exists(role_subparser)
+        self._add_role_options(role_parser)
+
+        return self
+
+    def _create_role_parser(self):
+        role_name = 'role - Manage roles'
+        role_usage = 'conjur [global options] role <subcommand> [options] [args]'
+
+        role_parser = self.resource_subparsers \
+            .add_parser('role',
+                        help='Manage roles',
+                        description=command_description(role_name,
+                                                        role_usage),
+                        epilog=command_epilog(
+                            'conjur role exists -k host -i hosts/myhost\t\t\t'
```

Comment:
Would it be better to use the combined `kind:identifier` instead of separate `-k` and `-i` flags? That's the way I opted for in #416, and it's also the way the Ruby CLI worked. I think the reason is that it's less clunky to have fewer CLI flags.